content/en/blog/2020/open-usage/index.md

```diff
@@ -10,7 +10,7 @@ Since [day one](/news/releases/0.x/announcing-0.1/), the Istio project has belie
 
 Istio is an open source project, released under the Apache 2.0 license. That means people can copy, modify, distribute, make, use and sell the source code. The only freedom people don't have under the Apache 2.0 license is to use the name Istio, or its logo, in a way that would confuse consumers.
 
-As one of the founders of the project, Google is the current owner of the Istio trademark. While anyone who is using the software in accordance with the license can use the trademarks, the historic ownership has caused some confusion and uncertainty about who can use the name and how, and at times this confusion has been a barrier to community growth. So today, as part of Istio’s continued commitment to openness, Google is announcing that the Istio trademarks will be transferred to a new organization, the Open Usage Commons, to provide neutral, independent oversight of the marks.
+As one of the founders of the project, Google is the current owner of the Istio trademark. While anyone who is using the software in accordance with the license can use the trademarks, the historic ownership has caused some confusion and uncertainty about who can use the name and how, and at times this confusion has been a barrier to community growth. So today, as part of Istio’s continued commitment to openness, Google is announcing that the Istio trademarks will be transferred to a new organization, the Open Usage Commons, to provide neutral, independent oversight of the marks, while we understand there are [concerns with the the new organization from IBM](https://developer.ibm.com/blogs/istio-google-open-usage-commons/).
```

Comment:
```suggestion
As one of the founders of the project, Google is the current owner of the Istio trademark. While anyone who is using the software in accordance with the license can use the trademarks, the historic ownership has caused some confusion and uncertainty about who can use the name and how, and at times this confusion has been a barrier to community growth. So today, as part of Istio’s continued commitment to openness, Google is announcing that the Istio trademarks will be transferred to a new organization, the Open Usage Commons, to provide neutral, independent oversight of the marks, while we understand there are [concerns with the new organization from IBM](https://developer.ibm.com/blogs/istio-google-open-usage-commons/).
```

Reply:
Consider moving to a new sentence.
`We understand there are [concerns with the new organization from IBM](https://developer.ibm.com/blogs/istio-google-open-usage-commons/) and others in the community.`

Reply:
Agree that it would read better as a separate sentence.